ReactApp/src/AppRoutes.jsx

```diff
@@ -0,0 +1,327 @@
+import ArchiverDataViewerDemo from "React-Automation-Studio/components/ArchiverDataViewer/ArchiverDataViewerDemo";
+import SettingsSteererXY from "React-Automation-Studio/components/SettingsPages/SettingsSteererXY";
+import SettingsSinglePS from "React-Automation-Studio/components/SettingsPages/SettingsSinglePS";
+import AdvancedSettingsSinglePS from "React-Automation-Studio/components/ExperimentalControlScreens/SettingsPages/AdvancedSettingsSinglePS";
+import Vault from "React-Automation-Studio/components/AlarmHandler/Vault";
+import AreaDetectorSimExample from "React-Automation-Studio/components/Examples/AreaDetectorSimExample";
+//staging
+import Staging from "./components/staging/Staging";
+import Example from "./components/staging/Example/Example";
+import Example1 from "./components/staging/Example/Example1";
+import Example2 from "./components/staging/Example/Example2";
+import Example3 from "./components/staging/Example/Example3";
+//system
+import Login from "React-Automation-Studio/components/SystemComponents/Login";
+import Probe from "React-Automation-Studio/components/SettingsPages/Probe";
+import Help from "React-Automation-Studio/components/docs/Help";
+import AppMain from "./AppMain";
+import AutomationStudioContext from "React-Automation-Studio/components/SystemComponents/AutomationStudioContext";
+import Administrator from "React-Automation-Studio/components/Administrator/Administrator";
+import UserProfile from "React-Automation-Studio/components/SystemComponents/userProfiles/UserProfile";
+import ProtectedRoute from "React-Automation-Studio/components/SystemComponents/ProtectedRoute";
+
+const AppRoutes = (props) => {
+  const context = useContext(AutomationStudioContext);
+  /* eslint-disable-next-line no-unused-vars */
+  const userData = context.userData;
+  /* eslint-disable-next-line no-unused-vars */
+  const roles = context.userData.roles;
+  /* eslint-disable-next-line no-unused-vars */
+  const username = context.userData.username;
+  let enableDemos =
+    typeof import.meta.env.VITE__DISABLE_DEMOS !== "undefined"
+      ? !(import.meta.env.VITE__DISABLE_DEMOS.toUpperCase() === "TRUE")
+      : true;
+
+  return (
+    <BrowserRouter>
+      <Routes>
+      <Route
+          exact
+          path="/"
+          element={
+            <ProtectedRoute>
+              <AppMain/>
+            </ProtectedRoute>
+          }
+        />
+       <Route
+          exact
+          path="/MainDashboard"
+          element={
+            <ProtectedRoute>
+              <MainDashboard />
+            </ProtectedRoute>
+          }
+        />
+        <Route
+          exact
+          path="/Administrator"
+          element={
+            <ProtectedRoute roles={["admin"]}>
+              <Administrator />
+            </ProtectedRoute>
+          }
+        />
+        {/* <ProtectedRoute path="/UserProfile" component={UserProfile} /> */}
```

Comment:
Can we add some comment to explain the purpose (e.g. "For restricted access use: ....").